src/mount/writedata.cc

```diff
@@ -199,9 +201,8 @@ struct DelayedQueueEntry {
 	}
 };
 
-} // anonymous namespace
-
 static std::atomic<uint32_t> maxretries;
+static std::atomic<uint32_t> gWaveTimeout;
```

Comment:
Consider gWriteWaveTimeout. I see the filename is writedata, but may help differenciating it from the read in searches.

Reply:
Applied.